src/amber/cli/templates/api/controller/granite/src/controllers/{{name}}_controller.cr.ecr

```diff
@@ -0,0 +1,73 @@
+class <%= class_name %>Controller < ApplicationController
+  def index
+    <%= Inflector.pluralize(@name) %> = <%= class_name %>.all
+    responds_with 200 do
+      json <%= Inflector.pluralize(@name) %>.to_json
+    end
+  end
+
+  def show
+    if <%= @name %> = <%= class_name %>.find params["id"]
+      responds_with 200 do
+        json <%= @name %>.to_json
+      end
+    else
+      responds_with 404 do
+        json {status: "not found"}.to_json
+      end
+    end
+  end
+
+  def create
+    <%= @name %> = <%= class_name %>.new(<%= @name %>_params.validate!)
+
+    if <%= @name %>.valid? && <%= @name %>.save
+      responds_with 201 do
+        json <%= @name %>.to_json
+      end
```

Comment:
These `responds_with` blocks seem verbose given that it's always going to respond with json, is that the only way to render json with Amber?

Reply:
Actually, shouldn't these be `respond_with` (singular)?

Reply:
still a WIP but yeah.